src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -951,7 +951,7 @@ sap.ui.define([
 		if (sGroupId) {
 			this._setProperty(this.mResourceGroupId, FHIRUtils.deepClone(aResPath), sGroupId, true);
 		}
-		if (!vServerValue && oRequestInfo.method === HTTPMethod.PUT) {
+		if (!this._isServerDateUpToDate(vServerValue, oResource, oRequestInfo.method)) {
```

Comment:
Method name should be `isServerStateUpToDate` isn't`?

Reply:
my bad,.. thanks!

Reply:
done